src/store/Store.js

```diff
@@ -157,6 +157,14 @@ class Store {
           portName: this.portName,
           payload: data
         }, null, (resp) => {
+          if (!resp) {
+            const error = chrome.runtime.lastError;
```

Comment:
Can we use the API from `getBrowserAPI()` ([here](https://github.com/tshaddix/webext-redux/blob/master/src/util.js#L6)) instead of assuming the environment is chrome? It should be available with `this.browserAPI` as it is initialized in the constructor.

Reply:
good catch 👍